index.js

```diff
@@ -39,6 +39,20 @@ Structures.extend('Guild', function(Guild) {
   return MusicGuild;
 });
 
+Structures.extend('Guild', function(Guild) {
```

Comment:
Will it break this feature if instead of extending the same class twice we would just rename MusicGuild and add twitchData to its constructor?

Reply:
apologize for the delay wanted to test 
i moved 
```     
 this.twitchData = {
        Interval: null,
        embedStatus: null,
        isRunning: false
      };
```
to the MusicData class and it works just fine

Reply:
whoops :D yeah, 
no need to extend the same class twice
Thank you for the heads up (that took me  a sec to fully understand what you were asking)

renamed - Class `MusicGuild` to `GuildData` (as it is more than just music now)
moved - `twitchData`  to the existing class 

Much Love
-Bacon